src/Altinn.Platform/IntegrationTest/StorageQueryTime.cs

```diff
@@ -0,0 +1,165 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Http;
+using Altinn.Platform.Storage.Models;
+using Altinn.Platform.Test.Integration.Fixtures;
+using Serilog;
+using Serilog.Core;
+using Xunit;
+
+namespace Altinn.Platform.Test.Integration
+{
+    /// <summary>
+    /// class to test query time.
+    /// </summary>
+    public class StorageQueryTime : IClassFixture<PlatformStorageFixture>
+    {
+        private readonly PlatformStorageFixture fixture;
+        private readonly HttpClient client;
+        public string instanceId;
+        private const string platformUrl = ""; // "http://platform.altinn.cloud";
+
+        private Logger logger = new LoggerConfiguration()
+        .WriteTo.Console()
+        .WriteTo.File("log.txt")
+        .CreateLogger();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PlatformStorageTests"/> class.
+        /// </summary>
+        /// <param name="fixture">the fixture object which talks to the SUT (System Under Test)</param>
+        public StorageQueryTime(PlatformStorageFixture fixture)
+        {
+            this.fixture = fixture;
+            this.client = this.fixture.Client;
+        }
+
+        /// <summary>
+        /// perform a cross partition query and measure time
+        /// </summary>
+        [Fact]
+        public async void CrossPartitionQueryApplicationOwner()
+        {
+            string[] owners = { "SKD", "BRREG", "NAV", "DIBK", "ACN", "OSLK", "BRMK", "DIFI", "ASKK" };
+
+            logger.Information("CrossPartitionQuery: ApplicationOwnerId");
+
+            foreach (string applicationOwnerId in owners)
+            {
+                Stopwatch watch = Stopwatch.StartNew();
+
+                string queryURI = platformUrl + "/api/v1/instances/query?applicationOwnerId=" + applicationOwnerId;
```

Comment:
platformUrl is an empty string, will this then work?

Reply:
Yes it will call the SUT's controller with empty platformUrl.